app/app.go

```diff
@@ -875,6 +876,24 @@ func (app *App) EndBlocker(ctx sdk.Context, req abci.RequestEndBlock) abci.Respo
 
 // InitChainer application update at chain initialization
 func (app *App) InitChainer(ctx sdk.Context, req abci.RequestInitChain) abci.ResponseInitChain {
+	// InitChainErrorMessage is the error message displayed when trying to sync testnet or mainnet from block 1 using the latest binary.
+	InitChainErrorMessage := "Unable to sync testnet or mainnet from block 1 using the latest version. " +
+		"Please use a snapshot to sync your node. " +
+		"Refer to the documentation for more information: " +
+		"https://www.zetachain.com/docs/nodes/start-here/syncing/"
```

Comment:
This can be refactored into a string literal to avoid concatenation:
```suggestion
	InitChainErrorMessage := `
Unable to sync testnet or mainnet from block 1 using the latest version.
Please use a snapshot to sync your node.
Refer to the documentation for more information:
https://www.zetachain.com/docs/nodes/start-here/syncing/`
```

Reply:
could also be a `const` right?